dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventsExportChangeLogsControllerTest.java

```diff
@@ -434,7 +460,11 @@ private static void assertChange(
         () ->
             assertEquals(dataElement.getUid(), actual.getChange().getDataValue().getDataElement()),
         () -> assertEquals(previousValue, actual.getChange().getDataValue().getPreviousValue()),
-        () -> assertEquals(currentValue, actual.getChange().getDataValue().getCurrentValue()));
+        () -> assertEquals(currentValue, actual.getChange().getDataValue().getCurrentValue()),
+        () ->
+            assertFalse(
```

Comment:
https://github.com/teleivo/dhis2-core/blob/9885bcbc879dc94d50b803919c543074b398234d/dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/JsonAssertions.java#L155-L161 might already do what you want